src/mui-setup.js

```diff
@@ -20,8 +20,12 @@ export default function muiSetup() {
       outline: 'none',
     };
   }
+  const themeConfig = {
+    ...adaptV4Theme(muiConfig),
```

Comment:
Do we still need this? I thought it was to still be able to use makeStyles. Is it because muiConfig still is on mui v4 "format"?

Reply:
You are right. I do not think I am capable to update the format of muiConfig to use V5 theme yet. Side effects could happen that I am not aware of. 
I will work on it in the future on a separate branch.